hibernate-core/src/main/java/org/hibernate/engine/jdbc/batch/JdbcBatchLogging.java

```diff
@@ -35,11 +35,7 @@ public interface JdbcBatchLogging extends BasicLogger {
 	Logger BATCH_LOGGER = Logger.getLogger( NAME );
 	JdbcBatchLogging BATCH_MESSAGE_LOGGER = Logger.getMessageLogger( MethodHandles.lookup(), JdbcBatchLogging.class, NAME );
 
-	@LogMessage(level = ERROR)
-	@Message(id = 100501, value = "Exception executing batch [%s], SQL: %s")
-	void unableToExecuteBatch(Exception e, String sql );
-
```

Comment:
Should this be deprecated instead? Not that I want to, but I see other messages (e.g. `CoreMessageLogger#unableToExecuteBatch`) got deprecated, and I have no idea why, so maybe there's a reason... ?

Reply:
I don't think we need to be deprecating methods of Logger APIs. We should simply delete them.

Any user-written program which is calling such a method is doing something wrong and deserves to break.

Reply:
@jrenaat see https://github.com/hibernate/hibernate-orm/pull/9044

I have deleted the useless `@Deprecated` methods.